docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+mysql:
+    extends:
+      file: common-compose.yml
+      service: mysql
+    links:
+      - consul:consul
+
+mysql_replica:
```

Comment:
This is giving us both some pain. Should we attempt to do leader election `onStart` to eliminate this? I've been avoid that because of risk of race conditions. Should we take it on now, though?